libs/awscdk/src/platform.ts

```diff
@@ -8,7 +36,62 @@ export class Platform implements platform.IPlatform {
   /** Platform model */
   public readonly target = "awscdk";
 
-  public newApp?(appProps: any): any {
+  public newApp(appProps: any): any {
     return new App(appProps);
   }
+
+  public newInstance(
+    type: string,
+    scope: Construct,
+    id: string,
+    ...args: any[]
+  ): any {
+    const Type = this.typeForFqn(type);
+    if (Type) {
+      return new Type(scope, id, ...args);
+    }
+
+    return undefined;
+  }
+
+  public typeForFqn(fqn: string): any {
```

Comment:
```suggestion
  public resolveType(fqn: string): any {
```

Let's rename this one.

Reply:
Done